src/webapp/src/auth/AuthenticationOptions.tsx

```diff
@@ -1,7 +1,68 @@
+import { Button } from '@small-ads/ui';
 import React from 'react';
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos';
+
+const LoginScreen = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: calc(100vh - 64px);
+`;
+
+const LoginMain = styled.div`
+  align-self: center;
+  flex: 1 1;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const LoginSign = styled.div`
+  font-size: 3rem;
+  letter-spacing: -0.066875rem;
```

Comment:
That looks strange. What is the purpose it serves?

Reply:
It makes the letter closer together so the sign  `Login to Small-ads` will be shorter:

Reply:
Okay fair then. Lets keep it. We only following the design in the end :)